pkg/ctl/create/cluster.go

```diff
@@ -92,26 +103,133 @@ func createClusterCmd(g *cmdutils.Grouping) *cobra.Command {
 	return cmd
 }
 
-func doCreateCluster(p *api.ProviderConfig, cfg *api.ClusterConfig, ng *api.NodeGroup, nameArg string) error {
+func doCreateCluster(p *api.ProviderConfig, cfg *api.ClusterConfig, ng *api.NodeGroup, nameArg string, cmd *cobra.Command) error {
```

Comment:
Nit: this is a very long function. Is there a way to split it in smaller ones? (in order to reduce scope & mental load)

Reply:
Yes, indeed! It would have to be re-factored very soon, e.g. to improve testing (see #393).

Reply:
I opened https://github.com/weaveworks/eksctl/issues/399.